### ISSUE TITLE
australia: from_date and until_date arguments not used

### ISSUE THREAD

**Issue by alecbass** [open-contracting/kingfisher-collect#668]
When running scrapy with the example arguments `scrapy crawl australia -a from_date=2016-01-01 -a until_date=2018-05-05`, the from_date and until_date arguments aren't actually used in the initial URL call when the scrapy request is made.

The URL itself calls a start date of 2004-01-01 no matter what the from_date argument happens to be, and until_date is the last day of the current year when the call is made.

This isn't a huge deal, but would be nice to fix. The workaround I've been using is this (kingfisher_scrapy/spiders/australia.py):

```
from datetime import date

import scrapy

from kingfisher_scrapy.base_spider import LinksSpider
from kingfisher_scrapy.util import parameters

class Australia(LinksSpider):

    name = "australia"

    default_from_date = "2004-01-01"

    # SimpleSpider
    data_type = "release_package"

    # LinksSpider
    next_page_formatter = staticmethod(parameters("cursor"))

    def start_requests(self):
        from_date = self.from_date.strftime("%Y-%m-%d") if self.from_date is not None else self.default_from_date

        until_date = self.until_date.strftime("%Y-%m-%d") if self.until_date is not None else f"{date.today().year}-12-31"

        url = (
            f"https://api.tenders.gov.au/ocds/findByDates/contractPublished/"
            f"{from_date}T00:00:00Z/{until_date}T23:59:59Z"
        )
        yield scrapy.Request(url, meta={"file_name": "start.json"})
```

I'm happy to open a pull request with this same change, unless it's something you'd rather do internally.

Thanks for reading :)

**Comment by alecbass** [comment]
Update: I've made a pull request here [https://github.com/open-contracting/kingfisher-collect/pull/669](https://github.com/open-contracting/kingfisher-collect/pull/669)

**Comment by yolile** [comment]
Hi @alecbass, we didn't support the `from_date` and `until_date` parameters for Australia yet, indeed. Thank you very much for your contribution, I will take a look at and integrate your changes

**Comment by yolile** [comment]
I've merged your PR and now you can use eg `scrapy crawl australia -a from_date=2016-01-01T00:00:00 -a until_date=2018-05-05T23:59:59`
Note that I've included the time too as the Australia's API support the time as well and some user may need that level of granularity when scraping the data. But you can set the dates as the example (from T00:00:00 to T23:59:59) to have the complete day data.
closed by #669

**Comment by alecbass** [comment]
Thank you so much @yolile!
